docs/intro.rst

```diff
@@ -426,6 +426,7 @@ This is the current list of error and warning codes:
 **(*)** In the default configuration, the checks **E121**, **E123**, **E126**,
 **E133**, **E226**, **E241**, **E242**, **E704**, **W503** and **W504** are ignored
 because they are not rules unanimously accepted, and `PEP 8`_ does not enforce them.
+Please note that if the option **--ignore=errors** is used, the default configuration will be overridden and ignore only the check(s) you want to skip.
```

Comment:
Please match the style of of the rest of the documentation here. We have line wraps.